orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -54,6 +56,20 @@ class ManualJudgmentStage extends LinearStage implements RestartableStage {
     return stage
   }
 
+  @Override
+  Optional<User> authenticatedUser(Stage stage) {
+    def stageData = stage.mapTo(StageData)
+    if (stageData.state != StageData.State.CONTINUE || !stage.lastModified?.user || !stageData.propagateAuthenticationContext) {
+      return Optional.empty()
+    }
+
+    def user = new User()
+    user.setAllowedAccounts(stage.lastModified.allowedAccounts)
```

Comment:
Certainly ... with fiat checks happening in clouddriver we would only need to inject the user and not the accounts.